pages/docs/concepts/asyncapi-document/reply-info.md

```diff
@@ -0,0 +1,149 @@
+
+In case of multiple requests made, each request is processed independently and sends the reply/response to the corresponding requestor when ready.
+
+The request/reply pattern in AsyncAPI works in the same fashion while supporting all the different sub-patterns. Irrespective of the sub-pattern you would like to implement, the Request/reply pattern can be implemented in the asyncAPI document using actions in the `Operation` object.
+
+You can add the reply info using the `Operation Reply` object under the `Operation` object. The `reply` object represents the response message.
+
+Here is diagram to illustrate the working of a basic request/reply pattern:
+```mermaid
+sequenceDiagram
+    participant Requester as Requester
+    participant Responder as Responder
+    Requester->>Responder: Send Request
+    activate Responder
+    Responder-->>Requester: Send Reply
+    deactivate Responder
+```
+
+## Sub-patterns
+
+### Dynamic response Channel
+There are situations where you do not know the reply channel at the design time. Instead, the reply channel is dynamically determined at runtime, based on factors such as the request message payload or header. 
+
+In this case, since you don't know what the address of the reply channel is yet, you can either assign null to the `address` property or omit the property entirely indicating that the address is not known yet. You can now use `Operation Reply Address` object to dynamically specify where the reply has to be sent. The `Operation Reply Address` object has a property called `location` which essentially takes in a runtime expression that specifies the address of the reply channel.
+
+```yml
+asyncapi: 3.0.0
+
+info:
+  title: Ping/pong example for dynamic reply channel sub-pattern
+  version: 1.0.0
+  description: Example with a requester that initiates the request/reply pattern where the reply will happen on whatever is defined in the header `replyTo` of the request.
+
+channels:
+  ping:
+    address: /ping
+    messages:
+      ping:
+        $ref: '#/components/messages/ping'
+  pong:
+    address: null
+    messages:
+      pong:
+        $ref: '#/components/messages/pong'
+
+operations:
+  pingRequest:
+    action: send
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
+      address:
+        description: The reply address is dynamically determined based on the request header `replyTo`
+        location: "$message.header#/replyTo"
+      channel: 
+        $ref: '#/channels/pong'
+  pongReply:
+    action: receive
+    channel: 
+      $ref: '#/channels/ping'
+    reply:
```

Comment:
I don't think it is good, this app has only ping operation. Pong is just a result of reply, there should not be a separate operation as this app is not receiving such info as pong by default

Reply:
So would you recommend I remove the pongReply operation?